src/string/src/lib.rs

```diff
@@ -323,21 +323,77 @@ pub unsafe extern "C" fn strstr(s1: *const c_char, s2: *const c_char) -> *mut c_
 }
 
 #[no_mangle]
-pub extern "C" fn strtok(s1: *mut c_char, s2: *const c_char) -> *mut c_char {
-    unimplemented!();
+pub extern "C" fn strtok(s1: *mut c_char, delimiter: *const c_char) -> *mut c_char {
```

Comment:
Would it be possible to replace this with a short function that just calls `strtok_r`, passing it a static?

Reply:
Sure that's probably fine. I'll test it out later.